include/PR/rcp.h

```diff
@@ -1,15 +1,98 @@
+#define VI_CTRL_GAMMA_DITHER_ON     0x00004 /* 2: default = on */
+#define VI_CTRL_GAMMA_ON            0x00008 /* 3: default = on */
+#define VI_CTRL_DIVOT_ON            0x00010 /* 4: default = on */
+#define VI_CTRL_SERRATE_ON          0x00040 /* 6: on if interlaced */
+#define VI_CTRL_ANTIALIAS_MASK      0x00300 /* [9:8] anti-alias mode */
+#define VI_CTRL_ANTIALIAS_MODE_1    0x00100 /* Bit [9:8] anti-alias mode */
+#define VI_CTRL_ANTIALIAS_MODE_2    0x00200 /* Bit [9:8] anti-alias mode */
+#define VI_CTRL_ANTIALIAS_MODE_3    0x00300 /* Bit [9:8] anti-alias mode */
+#define VI_CTRL_PIXEL_ADV_MASK      0x01000 /* [15:12] pixel advance mode? */
+#define VI_CTRL_PIXEL_ADV_1         0x01000 /* Bit [15:12] pixel advance mode? */
+#define VI_CTRL_PIXEL_ADV_2         0x02000 /* Bit [15:12] pixel advance mode? */
+#define VI_CTRL_PIXEL_ADV_3         0x03000 /* Bit [15:12] pixel advance mode? */
+#define VI_CTRL_DITHER_FILTER_ON    0x10000 /* 16: dither-filter mode */
+
+
 #define VI_NTSC_CLOCK       48681812        /* Hz = 48.681812 MHz */
 #define VI_PAL_CLOCK        49656530        /* Hz = 49.656530 MHz */
 #define VI_MPAL_CLOCK       48628316        /* Hz = 48.628316 MHz */
 
-#define DEVICE_TYPE_CART    0   /* ROM cartridge */
-#define DEVICE_TYPE_BULK    1   /* ROM bulk */
-#define DEVICE_TYPE_64DD    2   /* 64 Disk Drive */
-#define DEVICE_TYPE_SRAM    3   /* SRAM */
-#define DEVICE_TYPE_INIT    7   /* initial value */
+/**
+ * Audio Interface (AI) Registers
+ */
+#define AI_BASE_REG     0x04500000
+
+/* AI DRAM address (W): [23:0] starting RDRAM address (8B-aligned) */
+#define AI_DRAM_ADDR_REG    (AI_BASE_REG + 0x00)    /* R0: DRAM address */
+
+/* AI length (R/W): [14:0] transfer length (v1.0) - Bottom 3 bits are ignored */
+/*                  [17:0] transfer length (v2.0) - Bottom 3 bits are ignored */
+#define AI_LEN_REG          (AI_BASE_REG + 0x04)    /* R1: Length */
+
+/* AI control (W): [0] DMA enable - if LSB == 1, DMA is enabled */
+#define AI_CONTROL_REG      (AI_BASE_REG + 0x08)    /* R2: DMA Control */
+
+/* Value for control register */
+#define AI_CONTROL_DMA_ON   1   /* LSB = 1: DMA enable*/
+#define AI_CONTROL_DMA_OFF  0   /* LSB = 1: DMA enable*/
+
+/* 
+ * AI status (R): [31]/[0] ai_full (addr & len buffer full), [30] ai_busy 
+ *        Note that a 1->0 transition in ai_full will set interrupt
+ *           (W): clear audio interrupt 
+ */
+#define AI_STATUS_REG       (AI_BASE_REG + 0x0C)    /* R3: Status */
+
+/* Value for status register */
+#define AI_STATUS_FIFO_FULL (1 << 31)
+#define AI_STATUS_DMA_BUSY  (1 << 30)
+
+/* 
+ * AI DAC sample period register (W): [13:0] dac rate 
+ *   - vid_clock/(dperiod + 1) is the DAC sample rate
+ *   - (dperiod + 1) >= 66 * (aclockhp + 1) must be true
+ */
+#define AI_DACRATE_REG      (AI_BASE_REG + 0x10)    /* R4: DAC rate 14-lsb*/
+
+/* DAC rate = video clock / audio frequency
+ *   - DAC rate >= (66 * Bit rate) must be true
+ */
+#define AI_MAX_DAC_RATE 16384   /* 14-bit+1 */
```

Comment:
Above you have `DACRATE` and below you have `DAC_RATE`, are these og names?

Reply:
Yes they are